lux/core/series.py

```diff
@@ -55,3 +54,94 @@ def f(*args, **kwargs):
 
         f._get_axis_number = super(LuxSeries, self)._get_axis_number
         return f
+
+    def __repr__(self):
+        from IPython.display import display
```

Comment:
I'm wonder if we should reuse some components between the repr in LuxSeries and LuxDataFrame

Reply:
I believe for now we will hold off on consolidating anything, at least until we figure out #39